backendApi/src/main/java/com/virtuslab/gitmachete/backend/api/SyncToRemoteStatus.java

```diff
@@ -5,7 +5,11 @@
 @Data(staticConstructor = "of")
 public class SyncToRemoteStatus {
   public enum Relation {
-    Untracked, InSyncToRemote, AheadOfRemote, BehindRemote, DivergedFromAndNewerThanRemote, DivergedFromAndOlderThanRemote
+    NoRemotes, Untracked, InSyncToRemote, AheadOfRemote, BehindRemote, DivergedFromAndNewerThanRemote, DivergedFromAndOlderThanRemote
+  }
+
+  public static SyncToRemoteStatus noRemotes() {
+    return of(Relation.NoRemotes, "");
```

Comment:
While we are here, maybe it's more reasonable to make `remoteName` field @Nullable and assign `null` to it instead of empty string in situations like this?

Reply:
Right, makes sense. Done

Reply:
Btw hi @nullable, whoever you are ;)

Reply:
O, I even didn't notice that :rofl: 
I just forgot about `` around `@Nullable`, sorry...